README.md

```diff
@@ -330,6 +332,81 @@ extraObjects:
               restartPolicy: OnFailure
 ```
 
+## git-sync
+
+`extraObjects`, `extraContainers`, `extraInitContainers`, `extraVolumes`, and `extraVolumeMounts` can be combined to deploy git-sync. The following example relies on `emptyDir` volumes and works with `KubernetesExecutor`.
```

Comment:
This example does not use `extraObjects` right?

Reply:
Good point!